eidas-saml-parser/src/test/java/uk/gov/ida/notification/eidassaml/apprule/base/EidasSamlParserAppRuleTestBase.java

```diff
@@ -65,11 +65,11 @@
             ConfigOverride.config("connectorMetadataConfiguration.trustStore.password", truststore.getPassword())
     );
 
-    protected Response postEidasAuthnRequest(AuthnRequest eidasAuthnRequest) throws MarshallingException {
-        System.out.println(marshaller.transformToString(eidasAuthnRequest));
+    protected Response postEidasAuthnRequest(AuthnRequest authnRequest) throws MarshallingException {
+        System.out.println(marshaller.transformToString(authnRequest));
```

Comment:
need this here?

Reply:
Probably not. The tests aren't currently being used so could do with a tidy up when they are.